cmd/template.md

```diff
@@ -74,11 +78,11 @@ For more information on the project and to make contributions, visit the [GitHub
 {{ if .SecurityInsightsValue }}
 **Security Insights Value:** {{ .SecurityInsightsValue }}
 {{- end }}
-
+{{- end }}
 ---
 
-{{- end }}
-{{- end }}
+{{- end}}
+{{- end}}
```

Comment:
```suggestion
{{- end }}
{{- end }}
```

This makes me itch

Reply:
Oops, I was trying to fix other whitespace issues, and missed this.